contracts/internal/EverlongPositions.sol

```diff
@@ -169,13 +169,40 @@ abstract contract EverlongPositions is EverlongBase, IEverlongPositions {
     // │ Position Closing (Internal)                             │
     // ╰─────────────────────────────────────────────────────────╯
 
+    /// @dev Close positions until the minimum output is met.
+    /// @param _minOutput Output needed from closed positions.
+    function _closePositionsByOutput(uint256 _minOutput) internal {
```

Comment:
As mentioned elsewhere, I thought we all agreed to not closing any positions before maturity. Doing so significantly increases the complexity of dealing with things like sandwich attacks.

Reply:
Resolving since our stance on this has changed.